Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -66,6 +66,11 @@ class ChannelDTO: NSManagedObject {
                     $0.didChangeValue(for: \.id)
                 }
         }
+        
+        let newAt = lastMessageAt ?? createdAt
```

Comment:
The task says to move the calculation, but removing the calculation from `saveChannel(payload:)` was not possible as the parameter is required.

Reply:
Feel free to add some description of why is this here + some better naming for the variables would be cool, too